monkestation/code/modules/virology/living/spread_disease.dm

```diff
@@ -55,35 +55,3 @@
 
 		D.AddToGoggleView(src)
 	return TRUE
-
-/mob/dead/new_player/proc/DiseaseCarrierCheck(mob/living/carbon/human/H)
```

Comment:
keep this proc, because i do believe we could get this to work at some point in the future